src/client/pages/forum/Forum.jsx

```diff
@@ -1,26 +1,48 @@
 import Post from "./Post";
 import { useState, useEffect } from "react";
+import "../../styles/forum.css";
+import PostForm from "./forms/PostForm";
 
-
-const Forum = () => {
+const Forum = (props) => {
+    const {user} = props;
     const [posts, setPosts] = useState([]);
+    const [showForm, setShowForm] = useState(false);
+    const [newComment, setNewComment] = useState({});
 
     useEffect(() => {
         const fetchPosts = async () => {
-            const res = await fetch('http://localhost:4000/post')
-            const allData = await res.json()
-            setPosts(allData.data)
+            const res = await fetch('http://localhost:4000/post');
+            const allData = await res.json();
+            setPosts(allData.data);
         }
-        fetchPosts()
-    }, []);
+        fetchPosts();
+    }, [newComment]);
     
     console.log(posts);
 
 	return (
         <div className="forum">
-            {posts && posts.map((post, index) => {
-                return <Post key={index} post={post}/>
```

Comment:
I think react prefers key={post.id}, rather than index

Reply:
I fixed it